tutorpicasso/commands/run_extra_commands.py

```diff
@@ -0,0 +1,157 @@
+import re
+import subprocess
+from itertools import chain
+
+# Was necessary to use this for compatibility with Python 3.8
+from typing import Any, List
+
+import click
+from tutor import config as tutor_config
+
+COMMAND_CHAINING_OPERATORS = ["&&", "&", "||", "|", ";"]
+
+
+@click.command(name="run-extra-commands", help="Run tutor commands")
+def run_extra_commands() -> None:
+    """
+    This command runs tutor commands defined in PICASSO_EXTRA_COMMANDS
+    """
+    context = click.get_current_context().obj
+    tutor_conf = tutor_config.load(context.root)
+
+    picasso_extra_commands: Any = tutor_conf.get("PICASSO_EXTRA_COMMANDS", None)
+
+    if not picasso_extra_commands:
+        return
+
+    error_message = validate_commands(picasso_extra_commands)
+    if error_message:
+        raise click.ClickException(error_message)
+
+    list(map(run_command, picasso_extra_commands))
+
+
+def validate_commands(commands: Any) -> str:
+    """
+    Takes all the extra commands sent through config.yml and verifies that
+    all the commands are correct before executing them
+
+    Args:
+        commands (list[str] | None): The commands sent through PICASSO_EXTRA_COMMANDS in config.yml
+    """
+    splitted_commands = [
+        split_string(command, COMMAND_CHAINING_OPERATORS) for command in commands
+    ]
+    flat_commands_list: chain[str] = chain.from_iterable(splitted_commands)
+
+    invalid_commands = []
+    misspelled_commands = []
+    for command in flat_commands_list:
+        if "tutor" not in command.lower():
+            if find_tutor_misspelled(command):
+                misspelled_commands.append(command)
+            else:
+                invalid_commands.append(command)
```

Comment:
I'm guessing that if "tutor" is part of the command, it's considered correct. So it's not checked for misspelling or any other condition, right?
```suggestion
        if "tutor" in command.lower():
           continue
           
        if find_tutor_misspelled(command):
           misspelled_commands.append(command)
        else:
           invalid_commands.append(command)
```

Reply:
Yes! I applied your suggestion. Thanks.